node/playwright-wrapper/interaction.ts

```diff
@@ -117,9 +123,28 @@ export async function tab(
     return emptyWithLog(`Tab element: '${selector}' with options: '${options}'`);
 }
 
-export async function internalClick(selector: string, strictMode: boolean, options: string, state: PlaywrightState) {
+export async function internalClick(
+    selector: string,
+    strictMode: boolean,
+    options: string,
+    state: PlaywrightState,
+): Promise<boolean> {
     const locator = await findLocator(state, selector, strictMode, undefined, true);
-    await locator.click(JSON.parse(options));
+    try {
+        await locator.click(JSON.parse(options));
+        return true;
+    } catch (error: unknown) {
+        if (error instanceof Error) {
+            if (error.message.startsWith('locator.click: Target closed')) {
```

Comment:
i would think that the two if conditions can be combined by an `and` because in both else branches you throw.

And isn't the `else` unnecessary? Because you can either return if both conditions are met and otherwise just re-raise the error outside the if?!

Reply:
And i would change the order:

```javascript
 try {
      await locator.click(JSON.parse(options));
      return true;
  } catch (error: unknown) {
      if (!(error instanceof Error && error.message.startsWith('locator.click: Target closed'))) {
           throw error;
      }
      logger.warn('Supress locator.click: Target closed error');
      return false;  
  }